includes/class-api.php

```diff
@@ -1125,4 +1126,72 @@ private function fix_term_relationships( $post_id, $post_args ) {
 			),
 		);
 	}
+
+	/**
+	 * Maybe create terms that don't exist.
+	 *
+	 * While wp_set_object_terms does create terms that don't exist, we can't also insert
+	 * meta data such as slug and description, or termmeta.
+	 *
+	 * @since NEXT
```

Comment:
We don't have a method of resolving this using Viacom processes, do we?

Reply:
No, I resolve these manually when I merge to master and tag.